### DISCUSSION TITLE
Add system requirements - MacOS 13

### DISCUSSION THREAD

**Discussion by RandomUser123321432** [italomandara/CXPatcher#77] [Ideas]
It would be nice to add in ReadMe system requirements - MacOS 13.

**Comment by Mfarshore** [comment]
The system requirements should note that CXPatcher itself only runs under MacOS13, **_and_** that Crossover has to be running under Ventura as well to use the CXPatcher fix.
I patched Crossover under Ventura, then tried to run Crossover under Monterey, using Stray as a test. Under Monterey, Stray gives the Black Screen of Frustration (BSoF). Substituting the [nastys](https://github.com/nastys) libMoltenVK.dylib eliminates the BSoF when running Crossover under Monterey and Ventura, but is not necessary under Ventura as the CXPatcher fix eliminates the BSoF.
In summary, the CXPatcher fix for Crossover under Ventura eliminates the BSoF and improves the color as well, but under Monterey you still get the BSoF.
